src/Component/Modal/ApplicationInfo/ApplicationInfo.tsx

```diff
@@ -43,6 +47,10 @@ export const ApplicationInfo: React.FC<ApplicationInfoProps> = ({
         className="shogun-logo"
         src={defaultLogo}
       />
+      <Statistic
+        title="Admin Version"
```

Comment:
can you add i18n here?

for the german version I think `Admin-Version` should be correct